src/Kuzzle.ts

```diff
@@ -231,15 +231,12 @@ export class Kuzzle extends KuzzleEventEmitter {
     
     if (this._cookieAuthentication) {
       this.protocol.enableCookieSupport();
+      options.offlineMode = 'auto'; // Enable offline queue and replay
```

Comment:
Actually the offline mode should only be actived during the time the websocket is disconnected. (only if it was not already activated)

Reply:
Done

What has changed:
During the process of renewing the connection, the websocket now emit two new events that are used internally
`websocketRenewalStart` & `websocketRenewalDone`, the first one is emitted just before the disconnection and the other just after websocket connection has been restablished.

Kuzzle adds a listener to those events when cookieAuth is set to `true`, when the event `websocketRenewalStart` is triggered, Kuzzle will save the configuration state of the `autoReplay`, `autoQueue` and `autoResubscribe` and set all three to  `true`,
once the event `websocketRenewalDone` is triggered, Kuzzle will restore the previous state of the  `autoReplay`, `autoQueue` and `autoResubscribe` configurations